assessments/intake-process.md

```diff
@@ -0,0 +1,74 @@
+# Security Assessment Priorities & Pipeline Intake Process
+
+SIG-Security has a volunteer team of subject matter experts and industry
+professionals dedicated to helping SIG-Security members, the TOC, and the larger
+CNCF community maintain an understanding of the current state of security in the
+cloud native ecosystem and helping cloud native projects succeed.
+
+The following process describes how projects are prioritized for security
+assessments.
+
+# Authority
+
+Team members are welcome to submit PRs to streamline process when priorities
+are clear based on the criteria below. As needed, specific leaders (identified
+in the [repo root README](/README.md#security-assessments)) will coordinate
+the decision-making process.
+
+* The [Security Assessment
+  Facilitator](/governance/roles.md#security-assessment-facilitator) is
+  responsible for maintaining the assessment queue and may delegate
+  responsibilities to specific individuals by defining and filling documented
+  roles and/or inviting community participation.
+* A named chair provides oversight for the Security Assessment initiative,
+  responsible for liaising with the TOC: aligning prioritization with TOC needs
+  and goals by finding opportunities to highlight work of Security Assessment
+  team, resolving questions/concerns about prioritization, and serving as an
+  escalation point for projects or SIG members, if needed.
+
+# Pre-conditions
+
+* The project is either a CNCF project OR an assertion that the project is cloud
+  native (any objection must be resolved before an assessment would be
+  considered)
+* The project has identified a project lead and has a written self-assessment
+
+# Intake priorities
+
+1. TOC requests prioritization of a specific project.
+    * TOC request will not interrupt an ongoing assessment.
+    * TOC requests may jump the prioritized queue of projects waiting for an assessment.
+2. Projects that have received a CNCF Security Audit will be reviewed within a
+   year of audit. (For future audits, the security assessment will be a
+   pre-condition to the audit.)
+3. CNCF Projects that request a review (or invited by SIG members), prioritized
+   by project maturity (e.g. graduated projects will be highest priority, then
+   incubated projects, then sandbox).
+4. Non-CNCF Projects that request a review (or invited by SIG members).
+
+# Updates and renewal
+
+The Security Assessment team will aim to review assessed projects annually,
```

Comment:
This somewhat conflicts with 2 above.

Reply:
I changed this to "aim to" in response to your earlier comment.  does that help?